tsdb/chunkenc/histo.go

```diff
@@ -248,6 +249,15 @@ func (a *HistoAppender) Append(int64, float64) {}
 // * any buckets disappeared
 // * there was a counter reset in the count of observations or in any bucket, including the zero bucket
```

Comment:
Perhaps the special behavior for the StaleNaN should be described in the doc comment here.